tests/run_gen.py

```diff
@@ -44,26 +44,6 @@ def run_gen(fn: str, flags: int = 0, args: Optional[str] = None):
         "ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff0180"
         "80")
 
-    # this is the deserializer that's passed in to all blocks, from:
-    # https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/puzzles/chialisp_deserialisation.clvm.hex
-    deserializer = bytes.fromhex(
```

Comment:
Why is this going away?

Reply:
this was a misunderstanding by me. I was confusing running the generators directly (where one would have to pass in the deserializer) versus running the generator ROM and passing in the generator. In the latter case, the ROM contains the deserializer, and is responsible for calling the generator correctly.

This test only run the generators via the generator ROM.